base_search_fuzzy/models/ir_model.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+# © 2016 Eficent Business and IT Consulting Services S.L.
+# © 2016 Serpent Consulting Services Pvt. Ltd.
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import models, api
+from odoo.osv import expression
+
+
+_logger = logging.getLogger(__name__)
+
+
+def patch_leaf_trgm(method):
+    def decorate_leaf_to_sql(self, eleaf):
+        model = eleaf.model
+        leaf = eleaf.leaf
+        left, operator, right = leaf
+        table_alias = '"%s"' % (eleaf.generate_alias())
+
+        if operator == '%':
+
+            sql_operator = '%%'
+            params = []
+
+            if left in model._fields:
+                column = '%s.%s' % (table_alias, expression._quote(left))
+                query = '(%s %s %%s)' % (column, sql_operator)
```

Comment:
Hah more of just a note to remember to make this change. I was randomly digging around in the expression code for an unrelated thing, saw `column_format`, and remembered this code. Making the change now!